infra/pipelines/docker/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+########
+#
+# Run from git root's parent: with .env in local folder and ProjectDomino/ inside
+#
+# $ touch .env
+# $ sudo docker-compose -f ./ProjectDomino/infra/pipelines/docker/docker-compose.yml up -d prefect-agent
+#
+########
+
+version: '3.4'
+
+services:
+  prefect-agent:
+    image: neo4j:4.0.2-enterprise
+    build:
+      context: ../../../
+      dockerfile: ./infra/pipelines/docker/Dockerfile
+    container_name: prefect-agent
+    network_mode: 'bridge'
+    restart: unless-stopped
+    environment:
+      PREFECT__SERVER__HOST: ${PREFECT__SERVER__HOST:-http://host.docker.internal}
```

Comment:
Just discovered we should also set `PREFECT__SERVER__UI__HOST` and `PREFECT__SERVER__UI__PORT` - don't think it matters that much but without them the logged URLs will be incorrect (localhost instead of the actual host). @lmeyerov could you add them in this PR or I can add them later?